alerts/plugindefinition.yaml

```diff
@@ -37,6 +37,10 @@ spec:
       description: "SilenceTemplates are pre-defined silence templates that can be used to scheduled Maintenance Windows. The format consists of a list of objects including description, editable_labels (array of strings specifying the labels that users can modify), fixed_labels (map containing fixed labels and their corresponding values), status, and title."
       required: false
       type: list
+    - name: predefinedFilters
+      description: "PredefinedFilters are filters that are applied by default when the application is loaded. The format is a list of objects including name, displayname and matchers (containing keys corresponding value)."
+      required: false
```

Comment:
You should be aware to find a solution in the UI if no PredefinedFilters are set...

Reply:
not in this PR but yes.